### ISSUE TITLE
Intent to implement: amp-mowplayer

### ISSUE THREAD

**Issue by gopanisandip** [ampproject/amphtml#16696]
We want to implement component for video player supported by https://mowplayer.com

**Prodecure :**

	

- User will get media-id from here https://video.mowplayer.com/publisher/videos
- Add that media-id as attribute of amp-mowplayer.
- That's it now user can view videos from mowplayer and manage settings in mowplayer and that settings will be reflected in video.

**Example :** 

```
<amp-mowplayer 
	data-mediaid="meeqmaqg5js" 
	layout="responsive" 
	width="16" 
	height="9">
</amp-mowplayer>
```

This player support single video and playlist in single media-id attribute. so not required for anyother attribute
All our static files are on CDN

@aghassemi 
@mrjoro

**Comment by aghassemi** [comment]
@gopanisandip API is simple and LGTM, looking forward to the PR.

You also need a `postMesssage` API for your player that can be used to implement the [video-interface](https://github.com/ampproject/amphtml/blob/master/src/video-interface.js).

The video-interface is a common API that AMP expects all video players to implement. This allows AMP to treat all video players the same and to use it to provide common UX across players such as autoplay, dockable videos, managing playback across players, etc... The interface is fairly straightforward: common methods and events such as `play`, `pause`, `muted`, etc..

Please look at [amp-youtube.js](https://github.com/ampproject/amphtml/blob/master/extensions/amp-youtube/0.1/amp-youtube.js) for a reference implementation which uses `postMessage` to do the communication between the AMP component and the iframed video player to implement the interface. 

[amp-brid-player](https://github.com/ampproject/amphtml/blob/master/extensions/amp-brid-player/0.1/amp-brid-player.js) is another good reference for a layer that implements the interface.

**Comment by gopanisandip** [comment]
@aghassemi As per your guide i have added postMessage API for mowplayer.

**Comment by mrjoro** [comment]
Looks like this was implemented in #17306.  Please reopen (or file a new issue) if there's anything left to track.
